packages/@expo/cli/src/export/exportApp.ts

```diff
@@ -172,7 +173,10 @@ export async function exportAppAsync(
 
       // Generate SPA-styled HTML file.
       // If web exists, then write the template HTML file.
-      files.set('index.html', { contents: html });
+      files.set('index.html', {
+        contents: html,
+        targetDomain: 'client',
```

Comment:
Add a test to ensure this never copies the index.html to `client/index.html`. This shouldn't technically be possible due the targetDomain check, but if a server file slips in during SPA exporting, then it would create a corrupt output.

Reply:
That's a good point :+1: I'll make sure to check whether there's an appropriate static/SPA E2E test I can update. I haven't checked whether we've got a `files` test for `exportApp`, but I'll update what's there to ensure the files remain in the right place, if we don't have a test for it already 💯